apps/homepage/src/app/[lang]/page.tsx

```diff
@@ -1,13 +1,87 @@
 'use client'
 import styles from './home.module.css'
-import { useLocalization } from '../../lib/context/loc-context'
+import { motion, AnimatePresence } from 'framer-motion';
+import { useEffect, useState } from 'react';
+import { UiViewport } from '@uireact/foundation';
+import { renderDynamicPhrase } from './utils';
+
+const phraseData = {
+  phrase: "We {keyWords} the future while being {keyDescriptions}",
+  keyWords: ['{engineer}', '{create}', '{code}'],
+  keyDescriptions: ['{humans}', '{developers}', '{artists}']
```

Comment:
This looks way better I would just remove the `{}` around the words in your array as you won't need it there right? Just keep:

```
  phrase: "We {keyWords} the future while being {keyDescriptions}",
  keyWords: ['engineer', 'imagine', 'code'],
  keyDescriptions: ['humans', 'developers', 'artists']
```